### ISSUE TITLE
Make loading deprecated lua scripts seletable

### ISSUE THREAD

**Issue by minggo** [cocos2d/cocos2d-x#8627]
Modify `src/main.lua`，add these codes at begin

``` lua
CC_USE_DEPRECATED_API = true
CC_USE_STDLIB = true
require("cocos.init")
```

**Comment by minggo** [comment]
Template is modified in #8637.
lua-tests should be changed too.

**Comment by minggo** [comment]
lua-tests was fixed in #8644.

@samuele3hu 
Please connect issue to PR in future.
Thanks.
